tests/python/unittest/test_sparse_operator.py

```diff
@@ -2121,16 +2121,16 @@ def test_batchnorm_fallback():
         mean_std = [mx.nd.array(rolling_mean).tostype(stype), mx.nd.array(rolling_std).tostype(stype)]
 
         test = mx.symbol.BatchNorm(data, fix_gamma=True)
-        assertRaises(MXNetError, check_numeric_gradient, test, in_location, mean_std, numeric_eps=1e-2, rtol=0.16, atol=1e-2)
+        assertRaises(MXNetError, check_numeric_gradient, test, in_location, mean_std, numeric_eps=1e-3, rtol=0.16, atol=1e-2)
```

Comment:
For batchnorm training test it is still 1e-2. Is that test expected to be flaky too ?

Reply:
just curious why we need to increase numeric_eps for this fallback test when it works fine for the training test even on the same seed.

Reply:
The original training test passed more than 10000 trials on both CPU & GPU

Reply:
Maybe we can merge this to stablize the CI first, and I'll do a deeper dive of this later. How do you like that?

Reply:
merging. please track in #11758